### PR TITLE
Added help to get additional options for commands.

### LINKED ISSUE #1866: Better help on CLI
Current output from `-h` on CLI is:

```
$ newman -h
Usage: newman [options] [command]

Options:
  -v, --version               output the version number
  -h, --help                  output usage information

Commands:
  run [options] <collection>  URL or path to a Postman Collection.
```

It would be better to have more documentation here, something like:

```
$ newman -h
Usage: newman [options] [command]

Options:
  -v, --version               output the version number
  -h, --help                  output usage information

Commands:
  run [options] <collection>  URL or path to a Postman Collection.

Options:
  -e <source>, --environment <source>           Specify an environment file path or URL
  -g <source>, --globals <source>               Specify the file path or URL for global variables
  -d <source>, --iteration-data <source>        Specify a data source file (CSV) to be used for iteration as a path to a file or as a URL
  -n <number>, --iteration-count <number>       Specifies the number of times the collection has to be run when used in conjunction with iteration data file.
```

And so on. So, what is available in https://github.com/postmanlabs/newman#command-line-options.

### PR DESCRIPTION
**Pull request by InFog**
Added additional information to get help for commands.  Fixes #1866.

### CODE REVIEW

File: bin/newman.js [hunk 1]
```diff
@@ -80,6 +80,12 @@ program
         });
     });
 
+program.on('--help', function () {
+    console.info('');
+    console.info('To get available options for a command:');
+    console.info('  [command] -h');
```
Comment (kunagpal) [inline bin/newman.js]:
```suggestion
    console.info('  newman [command] -h');
```

File: bin/newman.js [hunk 2]
```diff
@@ -80,6 +80,12 @@ program
         });
     });
 
+program.on('--help', function () {
+    console.info('');
+    console.info('To get available options for a command:');
```
Comment (kunagpal) [inline bin/newman.js]: The above `console.info` statement can be removed if this one is prefixed with `\n`, like so:
```
console.info('\nTo...');
```

**Comment by InFog** [comment]
Applied the changes. Thank you, @kunagpal.

**Comment by kunagpal** [comment]
@InFog L.G.T.M, thanks for contributing to Newman! 😄

**Pull request merged**